install-cloud-sdk-build-task/task.json

```diff
@@ -86,6 +93,23 @@
       "helpMarkDown": "Check this to remove files from the target installation path."
     }
   ],
+  "dataSourceBindings": [
+    {
+      "target": "zone",
```

Comment:
The name of the input on this task is "Zone". It is capitalized here and not other tasks because of the difference in powershell vs typescript variable naming conventions.